test/integration/formservice/formservice.js

```diff
@@ -184,7 +188,8 @@ context('Formservice', function() {
       .its('search')
       .should('contain', 'filter[action.tags]=foo-tag')
       .should('contain', 'filter[flow]=1')
-      .should('contain', 'filter[patient]=1');
+      .should('contain', 'filter[patient]=1')
+      .should('contain', `filter[created]=<=${ createdAt }`);
```

Comment:
This would be URL encoded in a live scenario, right?

Reply:
<img width="206" alt="Screenshot 2024-02-08 at 10 50 09 PM" src="https://github.com/RoundingWell/care-ops-frontend/assets/2028470/71b476f6-14b9-4133-9f1d-848aca6424bb">

yeah the URL encode happens on the entities and then that `itsUrl` translates it back for the tests as well